core/src/main/scala/cats/collections/Set.scala

```diff
@@ -418,4 +424,9 @@ trait AvlSetInstances {
     override def eqv(x: AvlSet[A], y: AvlSet[A]): Boolean =
       iteratorEq(x.toIterator, y.toIterator)
   }
+
+  implicit def showSet[A: Show]: Show[AvlSet[A]] = new Show[AvlSet[A]] {
+    override def show(t: AvlSet[A]): String =
+      t.toIterator.map(_.show).mkString("AvlSet(", ", ", ")")
```

Comment:
Can you add a quick property test for this one, that compares this with `toString` for an `A` where `show` and `toString` are the same? (i.e. `Int` or similar)

Reply:
Sure, just added. For it to succeed, I had to change the output of the `Show` instance to match the `toString`, which outputs something like _Set(x,y,z)_ instead of _AvlSet(x, y, z)_ which I had before (and which would be consistent with cats-core). Any preference/constraint on what formatting to use?

Reply:
Sorry for the back-and-forth: I think what @LukaJCB meant was to change `toString` such that it _also_ produces `AvlSet(x,y,z)`.

Reply:
Luckily not a difficult commit - changed it to match the cats-core formatting.